samples/openthread/common/overlay-ot-defaults.conf

```diff
@@ -34,6 +34,7 @@ CONFIG_NET_IPV6_MLD=n
 # Stack sizes configuration
 CONFIG_NET_TX_STACK_SIZE=1200
 CONFIG_NET_RX_STACK_SIZE=1500
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1120
```

Comment:
Does it work with multiprotocol? I think at least 2KB were required if BT is in use.

config SYSTEM_WORKQUEUE_STACK_SIZE
	int
	default 2048 if BT_LL_SOFTDEVICE

Reply:
For the CLI I see only the config with the minimal multiprotocol configuration, which has the `CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024`. Changing to 2144.